Source/WebInspectorUI/UserInterface/Views/ComputedStyleDetailsPanel.js

```diff
@@ -267,6 +264,9 @@ WI.ComputedStyleDetailsPanel = class ComputedStyleDetailsPanel extends WI.StyleD
         variablesStyleSection.element.dir = "ltr";
         variablesStyleSection.style = style;
 
+        if (groupType !== WI.CSSStyleDeclaration.VariablesGroupType.Ungrouped)
+            variablesStyleSection.variablesGroupType = groupType;
```

Comment:
Why not just provide this on construction of the `WI.ComputedStyleSection`?  I feel like it should be the one to decide what to do with `WI.CSSStyleDeclaration.VariablesGroupType.Ungrouped`, not some UI class/logic.  This would also let you drop the `get variablesGroupType` and `set variablesGroupType`.

Reply:
Yeah, that's a good idea. Moved `variablesGroupType` as an option passed to the constructor. 

I intentionally didn't move other properties for `WI.ComputedStyleSection` to the new options object.
Perhaps I'll follow up on that in another patch. Trying to keep this one focused.